lib/cusparse/preconditioners.jl

```diff
@@ -80,29 +78,27 @@ for (bname,aname,sname,elty) in ((:cusparseScsric02_bufferSize, :cusparseScsric0
             if m != n
                 throw(DimensionMismatch("A must be square, but has dimensions ($m,$n)!"))
             end
-            info = csric02Info_t[0]
-            cusparseCreateCsric02Info(info)
+            info = IC0Info()
 
             function bufferSize()
                 out = Ref{Cint}(1)
                 $bname(handle(), m, nnz(A), desc, nonzeros(A), A.colPtr, rowvals(A),
-                       info[1], out)
+                       info, out)
                 return out[]
             end
             with_workspace(bufferSize) do buffer
                 $aname(handle(), m, nnz(A), desc,
-                        nonzeros(A), A.colPtr, rowvals(A), info[1],
+                        nonzeros(A), A.colPtr, rowvals(A), info,
                         CUSPARSE_SOLVE_POLICY_USE_LEVEL, buffer)
                 posit = Ref{Cint}(1)
-                cusparseXcsric02_zeroPivot(handle(), info[1], posit)
-                if posit[] >= 0
+                cusparseXcsric02_zeroPivot(handle(), info, posit)
+                if posit[] ≥ 0
```

Comment:
I'm not a fan of unnecessary Unicode uses like this, especially if font ligatures can accomplish the same locally.